templates/verify.ejs

```diff
@@ -1,7 +1,7 @@
-<h1>Thank You</h1>
+<h1>{{Thank You | globalize}}</h1>
 
 <p>
-  Thanks for registering. Please follow the link below to complete your registration.
+  {{Thanks for registering. Please follow the link below to complete your registration. | globalize}}
```

Comment:
Candy Ng [9:09 AM]  
Can you clarify this comment?
-------------
Sure.  Somebody needs to replace the {{   | globalize }} surrounded text with the translated text.  That "somebody" is the filter.  https://www.npmjs.com/package/strong-globalize#globalize-html-templates

How do you use the template?    Precompiled?    Please also note that since there are many variations in Html templating, *.html is the only file name strong-globalize supports.  Template globalization is one of the major portions of LoopBack globalization we need to look into.